tests/x11/enlightenment_first_start.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    my $self = shift;
+    assert_and_click "enlightenment_profile_selection";
+    assert_and_click "enlightenment_assistant_next";
+    assert_and_click "enlightenment_profile_size";
+    assert_and_click "enlightenment_assistant_next";
+    mouse_hide();
+    assert_screen "enlightenment_windowfocus";
+    assert_and_click "enlightenment_assistant_next";
+    mouse_hide();
```

Comment:
I had seen in my tests that mouse was 'out of the way', so it found the first 'next', and as long as I had to click another thing in between i twas fine - but for the end of the assistant it was just 'next' and the mouse kept on being there - making the needles no longer match